src/core/build_simulations.jl

```diff
@@ -80,17 +80,29 @@ function _build_stages(sim_ref::SimulationRef,
 
     for (k, v) in stages
         verbose && @info("Building Stage $(k)")
-        op_mod = OperationModel(DefaultOpModel, v.model, v.sys;
-                                optimizer = v.optimizer,
-                                parameters = true,
-                                verbose = verbose,
-                                kwargs...)
+        canonical = _build_canonical(v.model.transmission,
+                                    v.model.devices,
+                                    v.model.branches,
+                                    v.model.services,
+                                    v.sys,
+                                    v.optimizer,
+                                    verbose;
+                                    parameters = true,
+                                    kwargs...)
         stage_path = joinpath(sim_ref.models,"stage_$(k)_model")
         mkpath(stage_path)
-        write_op_model(op_mod, joinpath(stage_path, "optimization_model.json"))
+        write_op_model(canonical, joinpath(stage_path, "optimization_model.json"))
```

Comment:
If what this function really does is write the canonical model to disk then should it be renamed to "write_canonical_model"?  Or "write_model"?